invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Item/ItemForms/ItemEditor/ItemEditor.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Loader, Error } from '../../../../../common/components';
+import { ItemForm } from './components';
+
+export class ItemEditor extends Component {
+  componentDidMount() {
+    if (this.props.match.params.itemPid) {
+      this.props.fetchItemDetails(this.props.match.params.itemPid);
+    }
+  }
+
+  renderEditForm = pid => {
+    const { isLoading, error, data } = this.props;
+    return (
+      <Loader isLoading={isLoading}>
+        <Error error={error}>
+          <ItemForm
+            pid={pid}
+            data={data}
+            title="Edit item"
+            successSubmitMessage="Your item has been updated."
```

Comment:
nit: `Your item` or maybe just `The item has been updated`? This applies globally. @kprzerwa @topless ping, since you are working on UX

Reply:
`The item ${barcode} has been successfully updated.` ?

Reply:
I kept it simple and changed it to "The X was successfully created." and "The X was successfully updated." for all editor forms. Do you see any additional value in displaying the `barcode` for items? And what would we display for other record types?